README.md

```diff
@@ -69,16 +70,16 @@ There are some complications when building Flatpak to a different
 prefix than the system-installed version. First of all, the newly
 built Flatpak will look for system-installed flatpaks in
 `$PREFIX/var/lib/flatpak`, which will not match existing installed
-flatpaks. You can use `--with-system-install-dir=/var/lib/flatpak`
+installations. You can use `--with-system-install-dir=/var/lib/flatpak`
```

Comment:
"installed installations" sounds weird

Reply:
was meant to say "existing installations"...